### PR TITLE
add swapdims frontends.torch

### PR DESCRIPTION
**Pull request by PatternFinder**
link to issue: #2705

### CODE REVIEW

**Comment by PatternFinder** [comment]
question: since I've only edited the torch test functions, how come others' edits under the tensorflow test functions' lintting error is showing up on this PR? like so,

```
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py:260:1: W391 blank line at end of file
```

**Comment by sherry30** [comment]
Hi,

Those lint errors are on the upstream master branch so they are not caused by your PR, you don't have to worry about those. Your PR will get merged if the lint errors are not caused by your code 😄 
Also just leave a comment if you're done making changes so i know its ready to be reviewed,

Thanks

**Comment by PatternFinder** [comment]
@sherry30 Hi, I've fixed all the linting on my branch and I'm ready for review. 

for `test-frontend-torch / run-nightly-tests (pull_request)`, I'm seeing:
```
FAILED ivy/ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py::test_torch_swapdims[cpu-ivy.functional.backends.torch-True-False-torch_call-torch]
```
How should I resolve this?

File: ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py [hunk 1]
```diff
@@ -70,3 +71,61 @@ def test_torch_cat(
...
+):
+    input_dtype, value = dtype_value
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        container_flags=container,
+        instance_method=instance_method,
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py:124]: remove this

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

You're not supposed to pass `container_flags` and `instance_method` argument to the `test_frontend_function`

**Comment by PatternFinder** [comment]
Ok I removed the container_flags and instance_method args.

**Pull request closed without merging**